### PR TITLE
ES-852981 Customize  the Word to HTML 

### PR DESCRIPTION
**Pull request by Venkateshmuruganandam**
HI All,
I completed the Cross Platform code added in  Customize  the Word to HTML 

Task Link: <a href="https://dev.azure.com/EssentialStudio/Document%20Processing%20Libraries/_workitems/edit/852981">Add cross platform code for Customizing Word to HTML</a>

Regards
Venkatesh

### CODE REVIEW

File: File-Formats/DocIO/html.md [hunk 1]
```diff
@@ -289,10 +289,38 @@ You can customize the Word to HTML conversion with the following options:
 * Export the images as Base-64 embedded images
 * Omit XML declaration in the exported HTML file using [HtmlExportOmitXmlDeclaration](https://help.syncfusion.com/cr/file-formats/Syncfusion.DocIO.DLS.SaveOptions.html#Syncfusion_DocIO_DLS_SaveOptions_HtmlExportOmitXmlDeclaration).
 
-N> While exporting header and footer, DocIO exports the first section header content at the top of the HTML file and first section footer content at the end of the HTML file.
+N> 1. While exporting header and footer, DocIO exports the first section header content at the top of the HTML file and first section footer content at the end of the HTML file.
+N> 2. [HtmlExportImagesFolder](https://help.syncfusion.com/cr/file-formats/Syncfusion.DocIO.DLS.SaveOptions.html#Syncfusion_DocIO_DLS_SaveOptions_HtmlExportImagesFolder), [HtmlExportCssStyleSheetFileName](https://help.syncfusion.com/cr/file-formats/Syncfusion.DocIO.DLS.SaveOptions.html#Syncfusion_DocIO_DLS_SaveOptions_HtmlExportCssStyleSheetFileName) APIs are supported only in .NET Framework alone.
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/html.md:293]: The [HtmlExportImagesFolder](https://help.syncfusion.com/cr/file-formats/Syncfusion.DocIO.DLS.SaveOptions.html#Syncfusion_DocIO_DLS_SaveOptions_HtmlExportImagesFolder) and [HtmlExportCssStyleSheetFileName](https://help.syncfusion.com/cr/file-formats/Syncfusion.DocIO.DLS.SaveOptions.html#Syncfusion_DocIO_DLS_SaveOptions_HtmlExportCssStyleSheetFileName) APIs are only supported in the .NET Framework.

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: File-Formats/DocIO/html.md [hunk 2]
```diff
@@ -289,9 +289,36 @@ You can customize the Word to HTML conversion with the following options:
 * Export the images as Base-64 embedded images
 * Omit XML declaration in the exported HTML file using [HtmlExportOmitXmlDeclaration](https://help.syncfusion.com/cr/file-formats/Syncfusion.DocIO.DLS.SaveOptions.html#Syncfusion_DocIO_DLS_SaveOptions_HtmlExportOmitXmlDeclaration).
 
-N> While exporting header and footer, DocIO exports the first section header content at the top of the HTML file and first section footer content at the end of the HTML file.
+N> 1. While exporting header and footer, DocIO exports the first section header content at the top of the HTML file and first section footer content at the end of the HTML file.
```
Comment (florenceliliana) [inline File-Formats/DocIO/html.md:292]: N> 1. When exporting header and footer, DocIO exports the first section of header content at the top of the HTML file and the first section of footer content at the end of the HTML file.

**Review by florenceliliana — CHANGES_REQUESTED** [review]
The content review is completed.

File: File-Formats/DocIO/html.md (hunk 1 above)
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/html.md]: modified

File: File-Formats/DocIO/html.md (hunk 2 above)
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/html.md]: modified

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**